modules/product/commerce_product.module

```diff
@@ -339,12 +339,66 @@ function commerce_product_field_widget_form_alter(&$element, FormStateInterface
   $entity_type = $field_definition->getTargetEntityTypeId();
   $widget_name = $context['widget']->getPluginId();
   $required = $field_definition->isRequired();
-  if ($field_name == 'path' && $entity_type == 'commerce_product' && $widget_name == 'path') {
-    $element['alias']['#description'] = t('The alternative URL for this product. Use a relative path. For example, "/my-product".');
-  }
-  elseif ($field_name == 'title' && $entity_type == 'commerce_product_variation' && !$required) {
-    // The title field is optional only when its value is automatically
-    // generated, in which case the widget needs to be hidden.
-    $element['#access'] = FALSE;
+  switch (TRUE) {
```

Comment:
It's just a coding style replacing

    if(condition) {
    }
    elseif(condition){
    }

to

    case (condition):
    break;
    case (condition):
    break;

IMO it makes code more readable when there is a lot blocks better separating them visually. If you see "break" you'd know that this is the end of a block. Not the same with braces which are small and hard to detect sometimes in a complex code snippet.